app/src/ui/v_dashboard.py

```diff
@@ -1,15 +1,15 @@
 from typing import List, Optional, Tuple
 import datetime as dt
 import streamlit as st
-from snowflake.snowpark.context import get_active_session
+from snowflake.snowpark import Session
 
 from ui_common import warnings_banner
 from first_time_setup import get_is_first_time_setup_dismissed, \
     render as render_first_time_setup
 from chairlift_data import Machine, SensorType, get_machines, \
     get_sensor_types
 
-session = get_active_session()
+session = Session.builder.getOrCreate()
```

Comment:
IIRC there was a reason we had to switch to getOrCreate. Worth a comment since this is intended for teaching?

Reply:
This can work using streamlit run, but since it's not officially supported, we probably should not add the comment.